src/clr-icons/shapes/all.umd.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { AllShapes } from './all';
+
+export * from './all';
```

Comment:
Do we need this export?

Reply:
Technically no, except to make the svg builder script work.

Reply:
Are you asking for `all.umd.ts` specifically or `<setName>umd.ts` files in general? For each shape set, the export was necessary for our script that writes out the individual svg files.

Reply:
I see. That makes sense.